libcontainer/container_linux.go

```diff
@@ -1249,7 +1255,7 @@ func (c *linuxContainer) orderNamespacePaths(namespaces map[configs.NamespaceTyp
 			if strings.ContainsRune(p, ',') {
 				return nil, newSystemError(fmt.Errorf("invalid path %s", p))
 			}
-			paths = append(paths, p)
+			paths = append(paths, fmt.Sprintf("%s:%s", configs.NsName(nsType), p))
```

Comment:
> since the namespace type is already "known" (implicitly)

It isn't though. All we have is a path from the perspective of `nsenter`. And we need this for the code in #976 (we only can double fork if we've joined a PID namespace).